patterns/src/js/StoreNew.js

```diff
@@ -0,0 +1,33 @@
+export default class Store {
+	constructor(reducers){
+		this.subscribers = [];
+		this.reducers = reducers;
+		
+		this.state = {};
+	}
+	
+	dispatch(action){
+		this.state = this.applyReducers(this.state, action);
+		this.notifySubscribers();
+		
+		return action;
+	}
+	
+	subscribe(fn){
+		this.subscribers.push(fn);
+	}
+	
+	applyReducers(state, action) {
+		let reducerResult = this.reducers.activateLoadButtonReducer(state, action);
```

Comment:
Believe that store shouldn't know about activateLoadButtonReducer. Think that reducers should be passed as array.